modules/modal/css/lib/modal.scss

```diff
@@ -0,0 +1,26 @@
+@keyframes wdc-modal-bg-animation {
+  from {
+    background: none;
+  }
+  to {
+    background: rgba(0,0,0,0.65);
+  }
+}
+
+.wdc-modal-bg {
+  align-items: center;
+  animation-duration: 0.3s;
+  animation-name: wdc-modal-bg-animation;
+  background: rgba(0,0,0,0.65);
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100vw;
+}
+
+.wdc-modal {
+  @include wdc-popup-common();
```

Comment:
Can you replicate the animation from `.wdc-modal-animation-origin-bottom-center`? This should match the React modal